.gitignore

```diff
@@ -6,3 +6,4 @@ bin/
 # Node.js generated files for tree-sitter
 build/
 node_modules/
+node/
```

Comment:
Maybe the `node` binding is needed to run `tree-sitter generate` and `tree-sitter test`? I cannot run this command locally (after clone your branch):
```
PS:45> npx tree-sitter generate
npm ERR! could not determine executable to run

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\dongbow\AppData\Local\npm-cache\_logs\2023-10-09T21_07_05_532Z-debug-0.log
```